pkg/scheduler/profile/profile.go

```diff
@@ -39,19 +40,23 @@ type FrameworkFactory func(config.KubeSchedulerProfile, ...framework.Option) (fr
 type Profile struct {
 	framework.Framework
 	Recorder events.EventRecorder
+	Metrics  *metrics.PerProfile
```

Comment:
Does this mean the metrics cannot be leveraged by out-of-tree plugin developers? If so, IMO we should let the metrics managed by prometheus (k/component-base) library in a global manner.

(If we delegate the metric management to Prometheus, this is not a problem anymore)

Reply:
These are profile level metrics (attempts and latency of an entire extension point). Plugins don't need to report anything to them.